downloads/kr/github-git-cheat-sheet.md

```diff
@@ -2,49 +2,49 @@
 layout: cheat-sheet
 redirect_to: false
 title: GitHub Git Cheat Sheet
-byline: Git은 여러분의 노트북이나 데스크톱에서 GitHub을 사용할 수 있도록 해주는 오픈 소스 분산 버전 관리 시스템입니다. 해당 Cheat Sheet에서는 사용자가 빠르게 참고할 수 있도록 주로 사용되는 Git의 명령행(command line)을 간단히 요약하였습니다.
+byline: Git은 여러분의 노트북이나 데스크톱에서 GitHub을 사용할 수 있도록 해주는 오픈 소스 분산 버전 관리 시스템입니다. 이 Cheat Sheet는 자주 쓰이는 Git 명령을 쉽게 찾아볼 수 있게 정리하여 보여줍니다.
 leadingpath: ../../../
 ---
 
 {% capture colOne %}
 ## Git 설치하기
-GitHub은 가장 일반적인 저장소(repository)와 관련된 활동과 발전적 시나리오를 염두한 Git의 자동 갱신형 커맨드 라인 에디션을 위해 그래픽 사용자 인터페이스를 포함한 데스크톱 클라이언트를 제공합니다
+GitHub은 일반적인 저장소(repository) 관리 작업을 위해 그래픽 인터페이스 클라이언트와, 더 복잡한 작업을 위해 자동 업데이트 기능이 포함된 명령줄 인터페이스 Git 클라이언트를 제공합니다.
```

Comment:
GitHub은 일반적으로 많이 사용되는 저장소 관련 작업들을 위한 그래픽 데스크톱 클라이언트와 함께, 더 복잡한 시나리오를 위해 데스크탑 클라이언트를 자동으로 업데이트 하는 Git command line 에디션을 제공합니다

Reply:
I like the first part, but the second part seems out of sync with the [original text](https://github.com/github/training-kit/blob/master/downloads/github-git-cheat-sheet.md):

>(...) an automatically updating command line edition of Git for advanced scenarios.

Here, "automatically updating" seems to refer to the command line Git client itself, rather than the command line client being responsible for updating the graphical client. So I suggest a compromise:

"GitHub은 일반적으로 많이 사용되는 저장소 관련 작업을 위한 데스크톱 클라이언트와 함께, 더 복잡한 작업을 위해 자동으로 업데이트되는 Git command line 에디션을 제공합니다.

Reply:
yeah, 'automatically updating command line edition of Git' is quite ambiguous. 😅 Git CLI is not something from GitHub,, I thought it is the 'GitHub desktop command line tool' which is installed in /usr/local/bin/github, and we can use some cli of github,,

Reply:
Since there's no further background info found for `automatically updating command line edition of Git', I'm good on this.